app/src/main/java/com/globant/calculatoronboardingkotlin/mvp/view/CalculatorView.kt

```diff
@@ -22,8 +21,23 @@ class CalculatorView(activity: Activity) : ActivityView(activity), CalculatorCon
         inputLabel.text = operator
     }
 
-    override fun showError(enum: ErrorMessages) {
-        Toast.makeText(activity, enum.message, Toast.LENGTH_SHORT).show()
+    override fun showDivideByZeroError() {
+        Toast.makeText(activity, R.string.divide_by_zero_message, Toast.LENGTH_SHORT).show()
```

Comment:
PLs, in each show Error you call a Toast, may be you can implement a generic method to abstract this functionality

Reply:
thanks for the advice, added a new function to solve that issue